### DISCUSSION TITLE
page layout is not showing correctly

### DISCUSSION THREAD

**Discussion by ajaykmankani** [filamentphp/filament#13433] [Help] [answered]
### Package

Panel builder

### Package Version

v3.0

### How can we help you?

getting this error 
![image](https://github.com/filamentphp/filament/assets/53260795/ec08e9e7-e63a-4a77-bf9b-87c5dfecb47d)
and the panel look like this
![image](https://github.com/filamentphp/filament/assets/53260795/fd346b3a-deee-485b-9d99-901626362772)
so as you can see this is some js error which i dont know how it comes. the last change i have done is just put sortable() in the table `<?php

namespace App\Filament\Resources;

use App\Filament\Resources\ServiceProviderResource\Pages;
use App\Filament\Resources\ServiceProviderResource\RelationManagers;
use App\Livewire\ProviderPage;
use App\Models\Provider;
use App\Models\serviceProvider;
use Filament\Forms;
use Filament\Forms\Components\FileUpload;
use Filament\Forms\Components\Group;
use Filament\Forms\Components\Repeater;
use Filament\Forms\Components\RichEditor;
use Filament\Forms\Components\Section;
use Filament\Forms\Components\Select;
use Filament\Forms\Components\Textarea;
use Filament\Forms\Components\TextInput;
use Filament\Forms\Components\Toggle;
use Filament\Forms\Form;
use Filament\Forms\Set;
use Filament\Resources\Resource;
use Filament\Tables;
use Filament\Tables\Actions\BulkAction;
use Filament\Tables\Columns\ImageColumn;
use Filament\Tables\Columns\TextColumn;
use Filament\Tables\Columns\ToggleColumn;
use Filament\Tables\Table;
use Filament\Forms\Components\Builder;
use Filament\Forms\Components\ColorPicker;
use Illuminate\Database\Eloquent\Collection;
use Illuminate\Database\Eloquent\SoftDeletingScope;
use Illuminate\Support\Str;

class ServiceProviderResource extends Resource
{
    protected static ?string $model = ServiceProvider::class;

    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';

    public static function form(Form $form): Form
    {

        $existingServiceProviders = ServiceProvider::pluck('service_provider')->toArray();

        // Filter out the providers that are already in the ServiceProvider model
        $filteredProviders = Provider::whereNotIn('service_provider', $existingServiceProviders)
            ->pluck('service_provider', 'service_provider')
            ->toArray();
        return $form
            ->schema([

                Section::make('Service Provider Info')
                    ->schema([
                        Select::make('service_provider')
                            ->required()
                            ->options($filteredProviders)
                            ->searchable()
                            ->live(onBlur: true)
                            ->afterStateUpdated(function ($state, $set) {
                                $set('slug', Str::slug($state));
                            }),
                        TextArea::make('main_description')
                            ->required(),
                        TextInput::make('slug')
                            ->required()
                            ->disabled()
                            ->dehydrated()
                            ->unique(serviceProvider::class, 'slug', ignoreRecord: true),
                        TextInput::make('existing_number')
                            ->label('Existing Customer Number')
                            ->default('000000000')
                            ->required(),
                        TextInput::make('new_number')
                            ->label('New Customer Number')
                            ->default('000000000')
                            ->required(),
                        Repeater::make('tabs')
                            ->schema([
                                TextInput::make('tab_name')
                                    ->required(),
                                TextInput::make('heading')
                                    ->required(),
                                TextArea::make('description')
                                    ->required(),
                                TextInput::make('price')
                                    ->required(),
                                TextInput::make('months')
                                    ->required(),
                                TextInput::make('number')
                                    ->required(),
                            ])->cloneable(),
                        RichEditor::make('about')
                            ->required()
                            ->columnSpan('full'),
                        FileUpload::make('image')
                            ->label('Service Provider Logo')
                            ->columnSpanFull()
                            ->directory('provider_images')
                            ->image()
                            ->imageEditor(),
                        Toggle::make('active')
                            ->required()
                            ->default(1),

                    ]),
                Section::make('Plans & FAQ')
                    ->schema([
                        Repeater::make('plans')
                            ->schema([
                                TextInput::make('plan_name')
                                    ->required(),
                                TextInput::make('from')
                                    ->required(),
                                TextInput::make('months')
                                    ->required(),
                                Repeater::make('features')
                                    ->schema([
                                        TextInput::make('feature')
                                            ->required()
                                            ->columnSpan('full'),
                                    ]),
                                TextInput::make('number')
                                    ->default('000000000'),
                                TextInput::make('affiliate_link')
                                    ->url(),
                            ])->cloneable(),
                        Repeater::make('faq')
                            ->schema([
                                TextInput::make('question')
                                    ->required(),
                                TextInput::make('answer')
                                    ->required(),
                            ])->cloneable(),

                    ])->columns(2),

                Section::make('Web Page')
                    ->schema([
                        Builder::make('web_page')
                            ->blocks([
                                Builder\Block::make('color_scheam')
                                    ->schema([
                                        ColorPicker::make('color')
                                            ->label('Website Color Scheme')
                                            ->required(),
                                        ColorPicker::make('text_color')
                                            ->label('Text Color')
                                            ->required(),
                                    ]),
                                Builder\Block::make('navigation')
                                    ->schema([
                                        FileUpload::make('nav_icon')
                                            ->label('Image')
                                            ->directory('provider_images')

                                            ->imageEditor()
                                            ->image()
                                            ->columnSpanFull(),
                                        ColorPicker::make('nav_bar_color'),
                                        ColorPicker::make('nav_link_color'),
                                    ]),
                                Builder\Block::make('hero_section')
                                    ->schema([
                                        FileUpload::make('hero_image')
                                            ->label('Image')
                                            ->directory('provider_images')

                                            ->imageEditor()
                                            ->image()
                                            ->columnSpanFull(),
                                        TextInput::make('affiliate_link')
                                            ->url(),
                                        TextInput::make('phone_number')
                                            ->url(),
                                    ])
                                    ->columns(2),
                                Builder\Block::make('after_hero')
                                    ->schema([
                                        RichEditor::make('after_hero')
                                            ->label('After Hero Section')
                                            ->required(),
                                    ]),
                                Builder\Block::make('image')
                                    ->schema([
                                        FileUpload::make('url')
                                            ->label('Image')
                                            ->image()
                                            ->required(),
                                        TextInput::make('alt')
                                            ->label('Alt text')
                                            ->required(),
                                    ]),
                            ])
                    ])

            ]);
    }

    public static function table(Table $table): Table
    {
        return $table
            ->columns([
                TextColumn::make('id'),
                TextColumn::make('service_provider')
                    ->searchable()
                    ->sortable(),
                TextColumn::make('slug')
                    ->sortable(),
                ImageColumn::make('image')
                    ->label('Provider Logo')
                    ->sortable(),
                ToggleColumn::make('active')
                    ->sortable(),
                ToggleColumn::make('page')
                    ->sortable(),
            ])
            ->filters([
                //
            ])
            ->actions([
                Tables\Actions\EditAction::make(),
                Tables\Actions\DeleteAction::make(),
            ])
            ->bulkActions([
                Tables\Actions\BulkActionGroup::make([
                    Tables\Actions\DeleteBulkAction::make(),
                    BulkAction::make('active toggle')
                        ->action(function (Collection $records) {
                            foreach ($records as $record) {
                                $record->active = !$record->active;
                                $record->save();
                            }
                        })
                        ->icon('heroicon-m-sparkles')
                        ->requiresConfirmation('Are you sure you want to toggle active status?')
                        ->deselectRecordsAfterCompletion(),
                ]),
            ]);
    }

    public static function getRelations(): array
    {
        return [
            //
        ];
    }

    public static function getPages(): array
    {
        return [
            'index' => Pages\ListServiceProviders::route('/'),
            'create' => Pages\CreateServiceProvider::route('/create'),
            'edit' => Pages\EditServiceProvider::route('/{record}/edit'),
        ];
    }
}
`

**Comment by leandrocfe** [comment]
`php artisan about --only=Filament`
What is the Filament version?

**Reply by ajaykmankani** [reply]
Version ................................................................................................................................... v3.2.0

**Reply by leandrocfe** [reply, marked as answer]
Add `^3.2` in the composer.json and run `composer update -W`

**Reply by ajaykmankani** [reply]
thank you sir, its done, also sir for my knowledge can you pls explain what was the problem and how did your solution fix this issue, and also what type of issue this procedure can fix. Thank you again sir.

**Comment by ajaykmankani** [comment]
Version ................................................................................................................................... v3.2.0
